02-express-tutorial/controllers/people.js

```diff
@@ -0,0 +1,56 @@
+let { people } = require("../data");
+
+const getPeople = (req, res) => {
+    res.status(200).json(people);
+}
+  
+const addPerson = (req, res) => {
+    const { name } = req.body;
+    if (!name) {
+      return res.status(400).json({ success: false, message: "Please provide a name!" });
+    }
+    const newPerson = { id: people.length + 1, name };
```

Comment:
some pre-increment and post-increment reading if youre interested - https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Increment